cpp/include/cudf/quantiles.hpp

```diff
@@ -94,5 +94,29 @@ std::unique_ptr<table> quantiles(
   std::vector<null_order> const& null_precedence = {},
   rmm::mr::device_memory_resource* mr            = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Calculate approximate percentiles on an input tdigest column.
+ *
+ * tdigest (https://arxiv.org/pdf/1902.04023.pdf) columns are produced specifically
+ * by the TDIGEST and MERGE_TDIGEST groupby aggregations.  These columns represent
+ * compressed representations of a very large input data set that can be
+ * queried for quantile information.
+ *
+ * Produces a LIST column where each row N represents output from querying the
+ * corresponding tdigest from `input` row N. The length of each output list
+ * is the number of percentages specified in `percentages`.
```

Comment:
This is pretty nitpicky, but to me, `N` usually means a size. Using `i` for an index is more canonical. 
```suggestion
 * Produces a LIST column where each row `i` represents output from querying the
 * corresponding tdigest from `input` row `i`. The length of each output list
 * is the number of percentages specified in `percentages`.
```